src/TokenRequestFactory.js

```diff
@@ -1,38 +1,55 @@
 /* @flow */
 
-import { stringify } from 'query-string';
-
+import { URL } from 'url';
```

Comment:
Will this work in the browser? As I understand it, the browser will supply this implementation (if the browser supports it).

I think this can be verified by changing the [`jest.testEnvironment`](https://github.com/clubhouse/clubhouse-lib/blob/0faa296c024193f9e22f4d17f9ad4ad1b5fd652d/package.json#L48) in `package.json` to `jsdom`.

What do you think about using [`universal-url`](https://github.com/stevenvachon/universal-url) to handle this?